blackbox/docs/appendices/release-notes/4.0.3.rst

```diff
@@ -0,0 +1,65 @@
+.. _version_4.0.3:
+
+=============
+Version 4.0.3
+=============
+
+Released on 2019/08/06.
+
+.. NOTE::
+
+    Please consult the :ref:`version_4.0.0_upgrade_notes` before upgrading.
+    Before upgrading to 4.0.3 you should be running a CrateDB cluster that is
+    at least on 3.0.7.
+
+    We recommend that you upgrade to the latest 3.3 release before moving to 4.0.3.
+
+    If you want to perform a `rolling upgrade`_, your current CrateDB version
+    number must be at least :ref:`version_4.0.0`. Any upgrade from a version
```

Comment:
```suggestion
    number must be at least :ref:`version_4.0.2`. Any upgrade from a version
```

Earlier versions were a) not declared stable and b) had a bug preventing rolling uprades.

Reply:
ah ok thx, didn' t know that.